roc_example_test.go

```diff
@@ -36,3 +37,19 @@ func ExampleROC_weighted() {
 	// true  positive rate: [0 0.5 0.5 1 1 1 1]
 	// false positive rate: [0 0 0.25 0.25 0.5 0.75 1]
 }
+
+func ExampleROC_aUC() {
+	y := []float64{0.1, 0.35, 0.4, 0.8}
+	classes := []bool{true, false, true, false}
+
+	tpr, fpr := stat.ROC(0, y, classes, nil)
+	auc := integrate.Trapezoidal(fpr, tpr)
```

Comment:
Are people going to know this means "Area Under Curve"? I wouldn't have before this PR. Maybe add a comment above the call to Trapezoid?